madgraph/interface/reweight_interface.py

```diff
@@ -1756,72 +2254,72 @@ def create_standalone_directory(self, second=False):
             
             #multiparticles
             for name, content in self.banner.get('proc_card', 'multiparticles'):
-                try:
-                    mgcmd.exec_cmd("define %s = %s" % (name, content))
-                except madgraph.InvalidCmd:
-                    pass
-                    
-        if  second and 'tree_path' in self.dedicated_path:
-            files.ln(self.dedicated_path['tree_path'], path_me,name=data['paths'][0])
-            if 'virtual_path' in self.dedicated_path:
-                has_nlo=True
+                    try:
+                        mgcmd.exec_cmd("define %s = %s" % (name, content))
+                    except madgraph.InvalidCmd:
+                        pass
+                        
+            if  second and 'tree_path' in self.dedicated_path:
+                files.ln(self.dedicated_path['tree_path'], path_me,name=data['paths'][0])
+                if 'virtual_path' in self.dedicated_path:
+                    has_nlo=True
+                else:
+                    has_nlo=False
             else:
-                has_nlo=False
-        else:
-            has_nlo = self.create_standalone_tree_directory(data, second)
+                has_nlo = self.create_standalone_tree_directory(data, second)
 
-        if has_nlo and not self.rwgt_mode:
-            self.rwgt_mode = ['NLO']
+            if has_nlo and not self.rwgt_mode:
+                self.rwgt_mode = ['NLO']
```

Comment:
This is also not clear why you do this.
IN this section of the file, many change since to be re-indentation.
Some of them are quite worrying. 
If you did not want to change the code the best is likely to revert that section to what it was before.
(and maybe do another PR only with style change if you want).